tests/testthat/test-weekly.R

```diff
@@ -0,0 +1,721 @@
+# devtools::load_all()
+require(testthat)
+
+data("Flu2009")
+data("SARS2003")
+incid <- SARS2003$incidence
+
+dt <- 7L
+dt_vec <- c(2L,2L,3L)
```

Comment:
you manually repeat those vectors c(2L, 2L, 3L) etc throughout the test, can you pls use dt_vec and dt_vec_1 instead? I think it'll be easier to follow